tools/libaqr/vagrant.py

```diff
@@ -86,39 +86,43 @@ def shell(self, node: Optional[str], command: Optional[str]) -> int:
         return retcode
 
     @classmethod
-    def box_list(cls) -> List[str]:
-        """ List all known vagrant boxes, including unrelated to aquarium. """
+    def box_list(cls) -> List[Tuple[str,str]]:
+        """ List all known vagrant boxes with the provider, including unrelated to aquarium. """
 
         cmd = "vagrant box list --machine-readable"
         retcode, out, err = cls.run(None, cmd, interactive=False)
         if retcode != 0:
             raise VagrantError(msg=err, errno=retcode)
 
-        boxes = (name for name in out.splitlines()
-                 if name.count("box-name") > 0)
-        boxnames = [entry.split(",")[3] for entry in boxes]
-        return boxnames
+        boxes = [name for name in out.splitlines() if name.count("box-name") > 0]
```

Comment:
Instead of `name.count("box-name") > 0` use `'box-name' in name`, because:
1) better readability 
2) it is faster because does not try to count anything, I know there is no that much performance benefit but still, we need to use more efficient python way